cmd/client.go

```diff
@@ -0,0 +1,137 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/rpc/client"
+	"github.com/tendermint/tendermint/types"
+)
+
+var c = client.NewHTTP("tcp://localhost:44657", "/websocket")
+
+var getCmd = &cobra.Command{
+	Use:   "get",
+	Short: "Get data by key",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
+			panic("get expects 2 args: key, nonce")
+		}
+
+		type get struct {
```

Comment:
this wont work. our fields arent exposed! besides, just use the Tx struct we already have defined for the app ...